src/components/interaction-designer/flow-editors/import/LanguageAdder.vue

```diff
@@ -0,0 +1,253 @@
+<template>
+  <div>
+    <button
+      class="btn btn-primary"
+      @click="showAddLanguageModal">
+      {{ 'flow-builder.add-language' | trans }}
+    </button>
+    <b-modal
+      ref="add-language-modal"
+      title="Add Language"
+      :ok-title="'flow-builder.create' | trans"
+      :cancel-title="'flow-builder.cancel' | trans"
+      @ok.prevent="handleCreateLanguage">
+      <div class="form-group">
+        <validation-message
+          #input-control="{ isValid }"
+          message-key="language/new_language/label">
+          <text-editor
+            v-model="newLanguage.label"
+            :placeholder="'flow-builder.enter-label' | trans"
+            :label="'flow-builder.label' | trans"
+            :valid-state="isValid" />
+        </validation-message>
+      </div>
+      <div class="form-group">
+        <!-- From https://www.npmjs.com/package/iso-639-3 and use search input from resource editor -->
+        <validation-message
+          #input-control="{ isValid }"
+          message-key="language/new_language/iso_639_3">
+          <label class="form-check-label mt-2 mb-2 mr-2">{{'flow-builder.iso-639-3-label' | trans}}</label>
```

Comment:
And let's add `text-primary` class for these labels to have uniform styling with `<text-editor ` label:
- ISO 639 3 Code, ISO 3166 1 Locale (optional), 
- BCP 47 
- Generated from ISO 639 3, Locale, and Variant